### ISSUE TITLE
TypeError: Cannot read property 'inherits' of undefined

### ISSUE THREAD

**Issue by Randomfinger** [Reactive-Extensions/rx.angular.js#155]
Hi,
I included angular, rxjs and no error occured. When I include rx.angular I get the following Error:
Uncaught TypeError: Cannot read property 'inherits' of undefined
    at rx.angular.js:591
    at rx.angular.js:656
    at rx.angular.js:32
    at rx.angular.js:34

What can I do?

**Comment by paulpdaniels** [comment]
Which angular and rxjs versions?

**Comment by Randomfinger** [comment]
angularjs/1.6.4/angular.min.js
[..some directives etc.]
unpkg.com/rxjs@5.4.0/bundles/Rx.min.js 
rx.angular.js (also tried the .min.js)

**Comment by Randomfinger** [comment]
I solved the problem now by replacing the Rx code to this: https://raw.githubusercontent.com/Reactive-Extensions/RxJS/master/dist/rx.all.min.js

**Comment by paulpdaniels** [comment]
Yes [RxJS 5](https://github.com/ReactiveX/rxjs) is not supported by this library.
